client/karma.js

```diff
@@ -175,21 +147,8 @@ var Karma = function (socket, iframe, opener, navigator, location) {
     }
   }
 
-  var UNIMPLEMENTED_START = function () {
-    this.error('You need to include some adapter that implements __karma__.start method!')
-  }
-
-  // all files loaded, let's start the execution
-  this.loaded = function () {
-    // has error -> cancel
-    if (!hasError) {
-      this.start(this.config)
-    }
-
-    // remove reference to child iframe
-    this.start = UNIMPLEMENTED_START
-  }
-
+  // TODO: Does anyone use `this.store`???
+  //   https://gitter.im/karma-runner/karma?at=56e48e07618c335373eb497f
```

Comment:
To keep things simple, I think I will submit another PR to remove it exclusively